JSDemos/Demos/PivotGrid/LayoutCustomization/description.md

```diff
@@ -1 +1,12 @@
-The PivotGrid component supports multiple customizations allowing you to&nbsp;provide the look and feel that will satisfy any user. This demo illustrates only a&nbsp;small part of&nbsp;the available customizations. Toggle the check boxes above the PivotGrid to&nbsp;change the layout at&nbsp;runtime.
\ No newline at end of file
+PivotGrid allows you to customize its layout. This demo illustrates the following properties:
+
+- [rowHeaderLayout](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#rowHeaderLayout)            
+Allows you to specify layout type for the row header items. The *"standard"* layout displays child items on the right from the parent items. Assign "*tree*" to this property to display items more compactly as a hierarchical structure. 
+
+- [showTotalsPrior](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#showTotalsPrior)        
+Specifies whether to show the total rows and columns before or after data. Applies only if **rowHeaderLayout** is *"standard"*.
```

Comment:
I'm not sure I understand what is "before or after" data. What do you mean? A position at the top or bottom of your component or something else entirely?

Reply:
When this property is **false**:
![image](https://user-images.githubusercontent.com/33040934/133969451-9631088c-3eda-4188-9dbb-253c1f124362.png)

when it is **true**, row totals are displayed above the data rows and to the left of columns:
![image](https://user-images.githubusercontent.com/33040934/133969610-5afeaf65-1c15-4ade-96d4-b86b3eb15b33.png)


I will also change
>Applies only if **rowHeaderLayout** is *"standard"*.

to

>This property does not affect [row totals](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#showRowTotals) when **rowHeaderLayout** is *"tree"*.

Reply:
OK, I see. I'm not really sure how to say this better... Maybe something like ... totals precede or follow data cells...